DOCS-3667: Document annual billing feature for subscription billing model (#4447)
viamrobotics/docs: static/include/app/apis/generated/billing.md

### GetCurrentMonthUsage

Access data usage information for the current billing period for a given organization.

> For organizations with monthly billing, this returns current month usage. For organizations with annual billing, this returns usage within the current annual billing cycle.

Reviewer: ```suggestion
This method only returns usage for organizations with monthly billing at the end of the month (`"in_arrears": true`).
```
